src/handbook/company/communication.md

```diff
@@ -56,6 +56,20 @@ Slack to be matched automatically.
 
 ## Asynchronous communication
 
+Communication through different timezones or due to differences in availability
+requires knowledge to be persistent in sources that are approachable to others
+and non-ephemeral.
+
+Information should thus be stored in:
+1. Handbook
+1. GitHub issues
+1. Git Repositories pushed to GitHub
+1. Google Drive (Docs / Sheets / etc.)
+
+Don't store information in:
+1. Slack
+2. Figma
```

Comment:
I think there is a difference between slack and figma here. We store designs in figma as it provides interactivity you don't get with static screenshots to better communicate a design as part of an issue. None of the 'approved' list provide that functionality.

Capturing screenshots as point-in-time records to include in a GH issue is a useful practice to encourage ease of reference, but I don't think we should say we don't store things in figma.

Reply:
@knolleary I consider Figma like a paper notebook, it's great for ideation, but not an indexed and approachable store of information (for community members or some FF team members). Thus for any asset that needs delivery it should be stored in another system than Figma.

I think the difference between what you mean with what's OK in Figma is _intermediate_ deliveries. That's not knowledge or information, but on par with an unpublished branch for developers.

Reply:
Figma should be used for storing Designs - that's literally what it's designed for. We can share elsewhere, but to say that Figma can't be used to store Designs seems a very odd angle to take.

Reply:
> Thus for any asset that needs delivery it should be stored in another system than Figma.

That's not at all what it reads @joepavitt, it's for ideas. Delivery is always elsewhere, the actual product that users can use.

Reply:
All of our designs for Device Groups were prototyped in Figma for @Steve-Mcl to use. He and I have been going back and forth actively in comments in Figma, iterating the design, because it's the best place for it. Should we not be doing that?

Reply:
There is a distinction between short-term and long-term information storing as well as the different uses we have for the tool.

I think the key point of the proposed change is about where we store information in the long-term for future reference.

Figma is the right tool for doing the design work in, and being able to refer back to it as we iterate is important. That doesn't preclude including snapshots of 'agreed' designs in issues as a point-in-time reference.

However there are other uses of figma that wouldn't be appropriate to treat as the long-term store for reference. For example, where we use figjams to ideate on a topic - the output will typically result in something being updated in the docs/handbook elsewhere. The figjam isn't kept as the single source of truth.

There are parallels with Google Docs; some uses are entirely appropriate (as per our Info Security policies) and some will be more short-lived 'working' documents as part of the overall process

Reply:
@joepavitt AFAIK device groups were designed and the design is not being pursued, which makes it a great example why design and Figma are part of the ideation phase and hold little end-user benefit.

> Also not sure how Figma is different to Google Drive in this case?

I'll interpret it as a sarcastic comment for your benefit.

Reply:
> AFAIK device groups were designed and the design is not being pursued, which makes it a great example why design and Figma are part of the ideation phase and hold little end-user benefit.

They were designed, and Steve is building it now. https://github.com/FlowFuse/flowfuse/issues/2997